backend/app/services/source_network.py

```
@@ -260,6 +284,80 @@ def _merge_network_results(
+    request = Request(
+        url,
+        headers={
+            "Accept": "application/json",
+            "User-Agent": "DMARQ source intelligence",
+            **headers,
+        },
+    )
+    try:
+        with urlopen(
+            request, timeout=timeout
+        ) as response:  # nosec B310 - operator-configured endpoint.
```

Ruff (0.15.21)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)